tests/virt_autotest/Prj2_Host_Upgrade/host_upgrade_base.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+package host_upgrade_base;
+use strict;
+use warnings;
+use lib "/var/lib/openqa/share/tests/sle-12-SP2/tests/virt_autotest/lib";
+use base "teststepapi";
+use testapi;
+
+our $PRODUCT_TESTED_ON = "SLES-12-SP2";
+our $PROJECT_NAME = "Host_Upgrade";
+
+sub execute_script_run($$) {
+    my ($self, $cmd, $timeout) = @_;
+
+    my $ret = $self->local_string_output($cmd, $timeout);
+
+    if ($ret == 1 ) {
+        die "Timeout due to cmd run :[" . $cmd . "]\n";
+    }
+    return $ret;
+
+}
```

Comment:
I don't understand the purpose of this. Is it the same as `assert_script_run` from [here](https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L611)